packages/taucmdr/cf/software/installation.py

```diff
@@ -578,7 +578,7 @@ def configure(self, flags):
         LOGGER.debug("Configuring %s at '%s'", self.name, self._src_prefix)
         cmd = ['./configure', '--prefix=%s' % self.install_prefix] + flags
         LOGGER.info("Configuring %s...", self.title)
-        if util.create_subprocess(cmd, cwd=self._src_prefix, stdout=False, show_progress=True):
+        if util.create_subprocess(cmd, cwd=self._src_prefix, stdout=False, show_progress=True, env=None):
```

Comment:
I think this should be `env=env`. With `env=None` it will always pass in `None` for the environment even if one was passed in to `configure`

```suggestion
        if util.create_subprocess(cmd, cwd=self._src_prefix, stdout=False, show_progress=True, env=env):
```

Reply:
Oh whoops I thought I had changed that